recon/src/recon/tests.rs

```diff
@@ -315,17 +315,17 @@ fn word_lists() {
                     println!(
                         "\t{}: <- {}[{}]",
                         k,
-                        if response.keys.len() < 10 {
-                            format!("{}", response)
+                        if response.msg.keys.len() < 10 {
```

Comment:
I put this logic in the test but I wander if it should be in the Display functon for Recon insted?